server/config/routes.js

```diff
@@ -348,6 +348,8 @@ exports.configure = function configure(app) {
   /* Depot routes */
   app.get('/depots', depots.list);
   app.get('/depots/:uuid', depots.detail);
+  app.get('/depots/group/byUuids', depots.getDepotGroup);
```

Comment:
Why write an entirely new route?  Can't you just add the `uuids` parameter to the filters in the [/depots](https://github.com/IMA-WorldHealth/bhima/blob/master/server/controllers/inventory/depots/index.js#L113) route?

Otherwise, you'll need to write a couple new integration tests for this new route.